demo-ui/app/uk/gov/ons/addressIndex/demoui/controllers/SingleMatchController.scala

```diff
@@ -152,6 +161,65 @@ class SingleMatchController @Inject()(
       }
     }
   }
+
+  /**
+    * Perform match by calling API with address string. Can be called directly via get or redirect from form
+    * @param input
+    * @return result to view
+    */
+  def doGetUprn(input : String, page: Option[Int]) : Action[AnyContent] = Action.async { implicit request =>
```

Comment:
I don't think we need page/limit/offset for getByUprn method as it will return either one or no elements.